api/src/main/java/org/openmrs/ui/framework/UiUtils.java

```diff
@@ -397,9 +405,33 @@ public String formatDatePretty(Date date) {
 	 * @param date the date to format
 	 */
 	public String formatDatetimePretty(Date date) {
+		if (convertTimezones()) {
+			return toTimezone(date, getDatetimeFormat());
+		}
 		return formatDatePretty(date) + " " + DateFormatUtils.format(date, "hh:mm a", locale);
 	}
 	
+	/**
+	 * Formats a time, in the client timezone with the format in the GP_FORMATTER_TIME_FORMAT Change the
```

Comment:
For readability, you could either put a full stop before `Change the` or put it on a separate line.